bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/ItemTriggers.json

```diff
@@ -128,6 +138,40 @@
                "required":true
             },
             {  
+               "name":"oldState",
```

Comment:
ah, sorry, Github had it collapsed in an unpleasant way...
Ok then - but please change the "label" as well as at the moment it is identical to the "state" parameter.